console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx

```diff
@@ -215,40 +215,42 @@ export const ProjectListContainer: React.FC<IProjectListContainerProps> = ({
   if (isAllProjectSelected && selectedProjects.length !== projectList.length) {
     selectAllProjects(projectList);
   }
-  const handleOnSelectProject = (data: IProject, isSelected: boolean) => {
+
+  const handleOnSelectProject = (project: IProject, isSelected: boolean) => {
```

Comment:
can we try event name as always suggest event+postfix
handleOnSelectProject ==> onSelectProject